ark-activemq/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: {{ include "ark-activemq.fullname" . }}
+  labels:
+    {{- include "ark-activemq.labels" . | nindent 4 }}
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: {{ include "ark-activemq.fullname" . }}
+  minReplicas: {{ .Values.autoscaling.minReplicas }}
+  maxReplicas: {{ .Values.autoscaling.maxReplicas }}
+  metrics:
+    {{- if .Values.autoscaling.targetCPUUtilizationPercentage }}
+    - type: Resource
+      resource:
+        name: cpu
+        targetAverageUtilization: {{ .Values.autoscaling.targetCPUUtilizationPercentage }}
+    {{- end }}
+    {{- if .Values.autoscaling.targetMemoryUtilizationPercentage }}
+    - type: Resource
+      resource:
+        name: memory
+        targetAverageUtilization: {{ .Values.autoscaling.targetMemoryUtilizationPercentage }}
+    {{- end }}
+{{- end }}
```

Comment:
@fabricetriboix I think we should avoid auto scaling here unless we have clustering logic created in the deployment. 
When we have multiple broker with no clustering, each client would connect one of the available ones and other brokers/activemq pods would not know it as both are individuals.  For example if client connects to one broker and publishes a message to a topic, the other broker would not know about this.

Reply:
@naveen-armedia I agree with you and that's why I left a comment in the `values.yaml` file to not enable autoscaling. I felt I could leave the code in for when we want to implement it. But I agree it would be cleaner to remove that code, so I will remove it.